cmd/config.go

```diff
@@ -273,8 +273,8 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
 	return &servicesConfig, nil
 }
 
-func getAuthConfig(reader *bufio.Reader) (*v3.AuthConfig, error) {
-	authConfig := v3.AuthConfig{}
+func getAuthConfig(reader *bufio.Reader) (*v3.AuthnConfig, error) {
```

Comment:
Can this also be called getAuthnConfig also we need to create getAuthzConfig to ask user about authorization mode